.circleci/config.yml

```diff
@@ -61,3 +63,32 @@ jobs:
           command: |
             ./gradlew --stop
             LOMBOK_PROVIDED_BY=gradle-plugin ./gradlew --max-workers=2 compileJava
+
+  publish:
+    working_directory: ~/git-machete-intellij-plugin
+    docker:
+      - image: gitmachete/intellij-plugin-ci
+    steps:
+      - attach_workspace:
+          at: .
+      - deploy:
+          name: Publish plugin to Jetbrains Marketplace
+          command: ./gradlew publishPlugin
```

Comment:
I've already generated the token, I'm not filling it up it in CircleCI yet for a moment